.github/workflows/psalm-admin.yml

```diff
@@ -0,0 +1,32 @@
+name: Psalm analysis - Admin
+
+on:
+  push:
+    branches: [ "master", "main" ]
+  pull_request:
+    branches: [ "master", "main" ]
+    paths:
+      - 'service-admin/**'
+jobs:
+  psalm_admin:
+    name: Psalm PDF
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./service-admin/
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup PHP with tools
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools: vimeo/psalm
+      - name: Composer install
+        run: composer install --prefer-dist --optimize-autoloader --no-suggest --no-interaction --no-scripts
+      - name: Run psalm
+        run: psalm --output-format=github --taint-analysis --report=results.sarif
+      - name: Upload Security Analysis results to GitHub
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: ./service-admin/results.sarif
```

Comment:
Is this not relative to working directory?

Reply:
yeah, i rpesumed that oo, but it seemingly failed with just `./results.sarif`

happy to pop though a test though to see!